db/seeds.rb

```diff
@@ -16,10 +16,11 @@
     title: Faker::Adjective.positive,
     content: Faker::Quote.famous_last_words,
     receiver: Employee.create!(email: Faker::Internet.email(domain: "example#{i}.com"), password: "password"),
-    giver: Employee.create!(email: Faker::Internet.email(domain: "example#{i+1}.com"), password: "password")
+    giver: Employee.create!(email: Faker::Internet.email(domain: "example#{i+1}.com"), password: "password"),
+    company_value_id: CompanyValue.find_by(title: 'Honesty').id
```

Comment:
try running `rails db:seed:replant` and you will see that this is not working, because you are not creating any CompanyValue before trying to find one